src/Umbraco.Cms.Integrations.Commerce.Shopify/Constants.cs

```diff
@@ -15,7 +15,7 @@ public static class Constants
 
         public static class RenderingComponent
         {
-            public const string DefaultV8ViewPath = AppPluginFolderPath + "/Render/Products.cshtml";
+            public const string DefaultV8ViewPath = AppPluginFolderPath + "/RenderV8/Products.cshtml";
 
             public const string DefaultV9ViewPath = AppPluginFolderPath + "/Render/ProductsV9.cshtml";
```

Comment:
Would it make sense to change this path to also have the version in the folder name?
```suggestion
            public const string DefaultV9ViewPath = AppPluginFolderPath + "/RenderV9/Products.cshtml";
```